client/my-sites/theme/main.jsx

```diff
@@ -101,6 +103,34 @@ import ThemeStyleVariations from './theme-style-variations';
 
 import './style.scss';
 
+const LivePreviewButton = ( { siteSlug, themeId, themeType, isFullSiteEditingTheme } ) => {
```

Comment:
I guess this button should be visible only on simple sites as the BTP is not enabled on atomic sites

Reply:
Addressed this in https://github.com/Automattic/wp-calypso/pull/78456/commits/eaacc42e915ced3c730e6e53db88c58a2b592aa3.